tests/stress/sync_stress/test_sync.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+import time
+from pytest_testconfig import config as testconfig
+
+from tests.test_bs import add_multi_clients, get_conf, setup_bootstrap, save_log_on_exit
+import tests.queries as q
+from tests.convenience import convert_ts_to_datetime
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+SYNC_DONE = "sync done"
+START_SYNC = "start synchronize"
+
+
+def test_sync_stress(init_session, setup_bootstrap, save_log_on_exit):
+    bs_info = setup_bootstrap.pods[0]
+    cspec = get_conf(bs_info, testconfig['client'])
+
+    clients = add_multi_clients(init_session, cspec, 10)
+
+    hits = []
+    number_of_pods = len(clients) + 1  # add 1 for bootstrap pod
+    tts = 70
+    while len(hits) != number_of_pods:
+        print(f"waiting for all clients to finish downloading all files, sleeping for {tts} seconds")
+        time.sleep(tts)
+        hits = q.get_all_msg_containing(init_session, init_session, "Done downloading")
+
+    del cspec.args['remote-data']
+    cspec.args['data-folder'] = ""
+
+    # Adding a new client
+    res_lst = add_multi_clients(init_session, cspec, 1, 'client')
+    new_client = res_lst[0]
+
+    curr_try = 0
+    max_retries = 20
```

Comment:
please comment on the sizes and measures, why are we waiting for this long? shouldn't it be a function of the sample size?

Reply:
also perhaps you can print the sample size?

Reply:
I don't have the sample size in my code, it's transparent to the test, either way we'll have to change it manually with every sample swap.

Reply:
ok just leave a comment for which sample is used and whats its size... other wise we wouldn't know

Reply:
you can see it in the conf yaml file, I named the data folder as the number of days you recorded.
i'll add a comment as well